3p/ampcontext.js

```diff
@@ -177,6 +180,31 @@ export class AbstractAmpContext {
     return unlisten;
   };
 
+  /**
+   *  Send message to runtime requesting HTML snippet of the parent window.
```

Comment:
Sorry, missed this before.

A few doc nits:

1. Description leaks implementation details. How about `Requests HTML snippet from the parent window.`?
2. `{!Array<string>}`
3. How about replacing `an array of tag attributes to be left in the stringified HTML` with `Whitelist of attributes to be kept in the returned HTML string.`
4. `Callback to be invoked with the HTML string as the first parameter.`

Reply:
Done. How about now?